### DISCUSSION TITLE
How to get native eth/coin without bridging eth from l1 chain?

### DISCUSSION THREAD

**Discussion by dump0007** [ethereum-optimism/developers#539] [Deploying the OP Stack] [answered]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

I deployed a local instance of the op stack l2 rollup with sepolia testnet as the l1 chain and it works perfectly. I even bridged some eth from sepolia to the rollup and deployed some contracts. i want to know if there is a way we can configure to get native coin without bridging maybe by implementing some consensus or something like that.

**Comment by hash1go** [comment, marked as answer]
Hello,
Since OP Stack is an L2 chain, native token (ETH) must always be bridged from L1.
